teamspeak/stack.yml

```diff
@@ -1,24 +1,24 @@
 version: '3.1'
 services:
-    teamspeak:
-        image: teamspeak-server
-        restart: always
-        ports:
-          - 9987:9987/udp
-          - 10011:10011
-          - 30033:30033
-        environment:
-            TS3SERVER_DB_PLUGIN: ts3db_mariadb
-            TS3SERVER_DB_SQLCREATEPATH: create_mariadb
-            TS3SERVER_DB_HOST: db
-            TS3SERVER_DB_USER: root
-            TS3SERVER_DB_PASSWORD: example
-            TS3SERVER_DB_NAME: teamspeak
-            TS3SERVER_DB_WAITUNTILREADY: 30
-            TS3SERVER_LICENSE: accept
-    db:
-        image: mariadb
-        restart: always
-        environment:
-            MYSQL_ROOT_PASSWORD: example
-            MYSQL_DATABASE: teamspeak
+  teamspeak:
+    image: teamspeak-server
+    restart: always
+    ports:
+      - 9987:9987/udp
+      - 10011:10011
+      - 30033:30033
+    environment:
+      TS3SERVER_DB_PLUGIN: ts3db_mariadb
+      TS3SERVER_DB_SQLCREATEPATH: create_mariadb
+      TS3SERVER_DB_HOST: db
+      TS3SERVER_DB_USER: root
+      TS3SERVER_DB_PASSWORD: example
+      TS3SERVER_DB_NAME: teamspeak
+      TS3SERVER_DB_WAITUNTILREADY: 30
+      TS3SERVER_LICENSE: accept
+  db:
+    image: mariadb
```

Comment:
Did you want to pin this to a particular release channel (such as `mariadb:10.2`), or is `:latest` expected to always work?

Reply:
(this can be a follow-up PR if it needs to change)